draft-scenarios/clash/gold_rush.tex

```diff
@@ -17,123 +17,112 @@ \subsection*{\MakeUppercase{Player Setup}}
+  \item Certain locations have special rules, typically allowing players to acquire \textbf{Resource Cards} -- refer to the table below.
+  \item When defeating an enemy Main Hero, the victorious player may, instead of taking \mbox{5 \svg{gold}}, choose and take one Artifact from the opponent's Discard Pile.
+  \item A Hero carrying the Grail Token is not protected by the \textit{Sanctuary} effect.
+  \item If a Hero carrying the Grail Token surrenders or is defeated by an enemy Hero, they give the Token to the opponent.
+  \item If a Hero carrying the Grail Token is defeated by a Neutral Army, the Grail Token is placed on the Field where the Hero was defeated. Any Hero visiting this Field takes the Token and loses all remaining~\svg{movement}.
 \end{itemize}
 
 \subsection*{\MakeUppercase{Victory Conditions}}
 
 \begin{itemize}
-  \item At the end of any Round, a player/team who possesses the Grail Token and can afford all remaining \textbf{tranches} wins immediately.
-  \item In \textbf{PvP games}, if no player wins after Round 13, the winner is the player/team who is able to pay the highest \textbf{tranche} value. When calculating this final value, possession of the Grail Token is worth an additional 60 \svg{gold}.
+  \item At the end of any Round, your Hero possesses the Grail Token and your team can afford all remaining \textbf{tranches}.
+  \item In \textbf{PvP games}, if nobody wins after 13 Rounds, the winner is the player/team who is able to pay the highest \textbf{tranche} value with their remaining resources.
 \end{itemize}
 
+\columnbreak
+
 \subsection*{\MakeUppercase{Defeat Conditions}}
 
 \begin{itemize}
-  \item Any player/team unable to pay a required \textbf{tranche} loses the game immediately.
-  \item In \textbf{Solo play}, the player loses if they fail to meet the Victory Condition by the end of Round 13.
-  \item In \textbf{PvP games}, if all players/teams cannot pay their \textbf{tranche} in the same Round, the player/team closest to affording their \textbf{tranche} wins (with the Grail Token counting as 60 \svg{gold}).
+  \item Fail to pay a required \textbf{tranche}.
+  \item In \textbf{Solo play}: fail to meet the Victory Condition by the end of Round 13.
+  \item In \textbf{PvP games}, if both players/teams cannot pay their \textbf{tranche} in the same Round, the player/team with the highest \textbf{tranche} value in remaining resources wins, with the Grail Token counting as 60~\svg{gold}.
 \end{itemize}
 
-\vfill
-
 \subsection*{\MakeUppercase{Timed Events}}
 
-Players must pay tranches at the start of specific Rounds, as shown in the table below. Tranches due on Resource Rounds are paid \textbf{after} receiving resources and resolving all other Round-beginning effects.
+Players must pay tranches at the start of specific Rounds, as shown in the table below. After paying, \textbf{remove all Black Cubes} from every Windmill and Water Wheel on the map.
```

Comment:
Does it mean you pay before receiving resources, or it's implied that you pay after that?

Reply:
The rewritten rulebook says, that scenario timed events are after getting resources: https://github.com/Heegu-sama/Homm3BG/blob/main/sections/round_structure.tex#L21I assumed here it's redundant info.

But actually it says about "Events marked on the Round Tracker". I may bring back the clarification.